.github/workflows/plugin-sdk-version-check.yml

```
@@ -0,0 +1,48 @@
+name: Plugin SDK - Match version from html5 package with bigbluebutton.properties
+on:
+  push:
+    branches:
+      - "develop"
+      - "v2.[5-9].x-release"
+      - "v[3-9].*.x-release"
+  pull_request:
+    types: [opened, synchronize, reopened]
+permissions:
+  contents: read
+jobs:
+  plugin-sdk-version-check:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          
+      - name: Ensure jq is installed
+        run: sudo apt-get update && sudo apt-get install -y jq
+
+      - name: Merge branches
+        uses: ./.github/actions/merge-branches
+
+      - name: Extract plugin-sdk version from html5 
+        working-directory: bigbluebutton-html5
+        run: |
+          VERSION=$(jq -r '.dependencies["bigbluebutton-html-plugin-sdk"]' package.json | sed 's/^[^0-9]*//')
```

This approach doesn’t work because the SDK version might be specified as `~0.0.70`, which currently resolves to version `0.0.79`. However, the command would incorrectly interpret it as `0.0.70`.

It would be more accurate to use:

```bash
jq -r '.packages["node_modules/bigbluebutton-html-plugin-sdk"]["version"]' package-lock.json
```
```suggestion
          VERSION=$(jq -r '.packages["node_modules/bigbluebutton-html-plugin-sdk"]["version"]' package-lock.json | sed 's/^[^0-9]*//')
```